setup.cfg

```diff
@@ -7,3 +7,12 @@ requires = environment-modules, bash, python >= 2.6, python < 3
 
 [easy_install]
 
+[flake8]
+max-line-length = 120
+
+# Hound CI runs with Python 3 (no way around it),
+# so we need to specify some Python 2 builtins to avoid that it complains about them
+# cfr. https://stackoverflow.com/questions/47427916/how-to-config-hound-ci-to-support-python2-7
+builtins =
+    basestring,
+    reduce
```

Comment:
@FooBarQuaxx Why add this here too, only belongs in `.flake8.ini`?

Reply:
Sorry for the late response. `setup.cfg` according to the [documentation](http://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations) is one the default locations that `flake8` checks for configuration. I use `flake8` as part of the development workflow ( without specifying the exact location of the config file) so I've replicated that section from `.flake8.ini` into the `setup.cfg`. Since duplication is obviously not a good thing, I can (1) keep this change (since it's more standard) and remove the reference to `.flake8.ini` from the houndci config files or (2) keep using the current way.

Reply:
What happens if we change `.hound.yml` to use `setup.cfg` as `config_file` rather than using `.flake8.ini`?

If that doesn't cause trouble, let's make that change (maybe in a separate PR) and drop `.flake8.ini`?

Reply:
I've made that change in #2511, together with the pure stylistic changes in `easybuild/tools/options.py`

Once that's merged, please re-sync this with `develop`